core/src/main/scala/org/apache/spark/rdd/JdbcRDD.scala

```diff
@@ -32,7 +32,6 @@ private[spark] class JdbcPartition(idx: Int, val lower: Long, val upper: Long) e
   override def index: Int = idx
 }
 
-// TODO: Expose a jdbcRDD function in SparkContext and mark this as semi-private
```

Comment:
This is a very old TODO and we need to revisit if this is still necessary. Since Spark SQL is the new core, I'm not sure if anyone is interested in a new API to create jdbc RDD.

Reply:
So, could we remove this TODO comment ?

Reply:
yea